cowbird/handlers/impl/filesystem.py

```diff
@@ -15,6 +15,9 @@
 
 NOTEBOOKS_DIR_NAME = "notebooks"
 
+DEFAULT_UID = 1000
+DEFAULT_GID = 1000
```

Comment:
These are the values used in [birdhouse](https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/config/jupyterhub/jupyterhub_config.py.template#L115).

I change the ownership of the files to `1000:1000` to prepare them for access with birdhouse and jupyterhub.
Since it seems to be the only user used in birdhouse, I don't see another case where we would use a different user than the default.

I will add a comment here to explain that they come from birdhouse.

Reply:
I see. In that case, we should make these hardcoded values env vars in birdhouse (something like `USER_WORSPACE_UID`, `USER_WORKSPACE_GID`), which would be injected in the jupyter notebook template configuration. The same variables would be injected in `cowbird`'s service `env` list, with something like `COWBIRD_FILESYSTEM_UID = "${USER_WORKSPACE_UID}`. The code here in cowbird would do `os.getenv("COWBIRD_FILESYSTEM_UID", 1000)`. 
This will make the mapping more explicit because of the `USER_WORSPACE_UID`/`USER_WORSPACE_GID` shared between the two services.

Reply:
I agree that this would make much more sense. I think I was reticent to change `birdhouse` since the `1000:1000` was used in some scripts that I have no knowledge of in `birdhouse`.

Do you have a preference between updating this in a separate PR, or should I include this change in the existing PR on `birdhouse`?

Reply:
You can prepare ahead the `os.getenv` part in this PR. 
When it will be time to include this new version of cowbird in birdhouse, then you can apply the other changes to align them.